Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'rails', '~> 4.2.11.1'
 ## Gems in Alphabetical Order
 gem 'activerecord-session_store'
 gem 'activesupport', '~> 4.2.11.1'
-gem 'airbrake'
+gem 'airbrake', '~> 9.5'
```

Comment:
I think these changes should be undone, because these  gemfile changes are not needed by everyone.